scripts/generateAssetData/arbitrum/index.ts

```diff
@@ -23,7 +24,10 @@ export const getAssets = async (): Promise<Asset[]> => {
   // Regular Portals assets however, should be last, as Coingecko is generally more reliable in terms of e.g names and images
   const [portalsPools, portalsAssets] = partition(_portalsAssets, 'isPool')
   const allAssets = uniqBy(
-    portalsPools.concat(assets).concat(portalsAssets).concat([arbitrum]),
+    portalsPools
+      .concat(assets)
+      .concat(portalsAssets)
+      .concat([unfreeze(arbitrum)]),
```

Comment:
blocking (same for other chains in this `scripts/generateAssetData/`): from a type level, all `unfreeze()` does is removing the `Readonly<T>` constraint, but `Readonly<Asset>[]` is actually compatible with `Asset[]`:

```suggestion
portalsPools.concat(assets).concat(portalsAssets).concat(arbitrum),
```

Reply:
No this actually crashes the asset generation script because it tries to mutate the frozen object (when setting `relatedAssetKey`. The base assets were frozen for paranoia because insanity can follow if we allow it - this forces us to be explicit when mutating base assets be calling `unfreeze` which creates a shallow clone.